routers/repo/attachment.go

```diff
@@ -123,6 +123,19 @@ func GetAttachment(ctx *context.Context) {
 		}
 	}
 
+	//If we have a signed url (S3, object storage), redirect to this directly.
+	u, err := storage.Attachments.URL(attach.RelativePath(), attach.Name)
```

Comment:
We should have a setting item from the `app.ini` here because the s3 service maybe an internal service which is not public.

Reply:
Added a "ServeDirect" option to allow this/future storage types to serve files directly instead of using Minio